tests/test_coin_combination.py

```diff
@@ -1,3 +1,16 @@
+import pytest
 from app.coin_combination import get_coin_combination
 
-# write your tests here
+
+@pytest.mark.parametrize(
+    "cents,expected_coin_combination",
+    [
+        (0, [0, 0, 0, 0]),
```

Comment:
Please, use `pytest.param` and `id` to provide a description of what you check in each test. Example:
```
pytest.param(
            0,
            [0, 0, 0, 0],
            id="should return all zeroes when variable `cents` is equal to zero",
        ),
```